common/reviews/api/gulp-core-build.api.md

```diff
@@ -179,6 +180,7 @@ export interface IExecutable {
     execute: (config: IBuildConfig) => Promise<void>;
     getCleanMatch?: (config: IBuildConfig, taskConfig?: any) => string[];
     isEnabled?: (buildConfig: IBuildConfig) => boolean;
+    maxBuildTime?: number;
```

Comment:
The name should indicate the units, e.g. `maxBuildTimeMs` or `maxBuildTimeSeconds`.

Reply:
Or just `maxBuildMs`